ha-neopool-mqtt-package.yaml

```diff
@@ -406,14 +406,14 @@ mqtt:
     - unique_id: "neopool_mqtt_filtration_switch"
       name: "NeoPool MQTT Filtration Switch"
       device_class: switch
-      command_topic: "cmnd/SmartPool/Backlog"
+      command_topic: "cmnd/SmartPool/NPFiltration"
       state_topic: "tele/SmartPool/SENSOR"
       value_template: >-
         {{ value_json.NeoPool.Filtration.State }}
       optimistic: false
       retain: false
-      payload_on: "NPFiltration 1;Delay 30;Teleperiod"
-      payload_off: "NPFiltration 0;Delay 30;Teleperiod"
+      payload_on: "1"
```

Comment:
looking at the json, all payload_on / off with 0 and 1 are int. So they shouldn't be quoted. See also state_on / off.

Reply:
I stand corrected, it has to be a string, but it worked also as int. I guess HA internally converts everything to a string.